### PR TITLE
Scenes: Enable live now feature

### PR DESCRIPTION
**Pull request by kaydelaney**
Enables the live now feature on scenes dashboards and allows for toggling it on/off in dashboard settings.

### CODE REVIEW

File: public/app/features/dashboard-scene/settings/GeneralSettingsEditView.tsx [hunk 1]
```diff
@@ -68,6 +68,10 @@ export class GeneralSettingsEditView
     return dashboardSceneGraph.getCursorSync(this._dashboard);
   }
 
+  public getLiveNowTimer(): behaviors.LiveNowTimer {
+    return sceneGraph.findObject(this._dashboard, (s) => s instanceof behaviors.LiveNowTimer) as behaviors.LiveNowTimer;
```
Comment (ivanortegaalba) [inline public/app/features/dashboard-scene/settings/GeneralSettingsEditView.tsx:72]: I really dislike this from TS :( 

We can avoid casting here if we contemplate the behavior could be `undefined` and in `onLiveNowChange` we check if it is an instance of `behaviors.LiveNowTimer`.

File: public/app/features/dashboard-scene/settings/GeneralSettingsEditView.tsx [hunk 2]
```diff
@@ -132,8 +136,13 @@ export class GeneralSettingsEditView
...
 
-  public onLiveNowChange = (value: boolean) => {
-    // TODO: Figure out how to store liveNow in Dashboard Scene
+  public onLiveNowChange = (enable: boolean) => {
+    try {
+      const liveNow = this.getLiveNowTimer();
+      enable ? liveNow.enable() : liveNow.disable();
+    } catch (err) {
+      console.error(err);
+    }
```
Comment (ivanortegaalba) [inline public/app/features/dashboard-scene/settings/GeneralSettingsEditView.tsx:145]:
```suggestion
      const liveNow = this.getLiveNowTimer();
      if(liveNow instanceof behaviors.LiveNowTimer){
            enable ? liveNow.enable() : liveNow.disable();
      } else {
          throw new Error('Live now timer cannot be found.')
      }
    }
```

**Review by ivanortegaalba — COMMENTED** [review]
Let's add some tests here! But looks good so far!

**Review by dprokop — CHANGES_REQUESTED** [review]
I think read-only wise this works great, but the live now option is not detected and persisted when changed.

**Review by dprokop — APPROVED** [review]
LGTM

**Review by ivanortegaalba — APPROVED** [review]
💯

**Pull request merged**